tests/test_profile.py

```diff
@@ -235,3 +235,50 @@ def test_that_links_in_the_services_page_return_200_code(self, base_url, seleniu
                 bad_urls.append(check_result)
 
         assert 0 == len(bad_urls), u'%s bad links found. ' % len(bad_urls) + ', '.join(bad_urls)
+
+    @pytest.mark.credentials
+    @pytest.mark.nondestructive
+    def test_that_user_can_view_external_accounts(self, base_url, selenium, vouched_user):
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+        settings = home_page.header.click_settings_menu_item()
+
+        irc_form = settings.external_accounts.irc_form
+        external_accounts_form = settings.external_accounts.external_accounts_form
+        assert irc_form
+        assert external_accounts_form
```

Comment:
These asserts are just checking that the page objects are returning something thruthy. I suspect you actually want to verify that these elements are displayed. Adding an `is_displayed` to each page region object might be a good approach. Also, there's not really any need to create variables for these, we can just use them directly.